### ISSUE TITLE
Date field picker limited to +/- 10 years

### ISSUE THREAD

**Issue by texnixe** [getkirby/kirby#2480]
**Describe the bug**  
The date field year picker is limited to +/- 10 years which is an issue when you want to enter dates like birthdays. While it is possible to set a min date, this only has an effect on the year selects in the input, while the picker still only shows a 20 year range and you have to go in 10 year steps backward and forward.

**To Reproduce**  
1. Try to enter a date more than 10 years in the past in a date field.
2. Without setting a min date, this is not possible at all.
3. When adding a min option, at least the input select shows the full year range, but the year selector inside the picker doesn't.

**Expected behavior**  
The date picker selector should show the same year range as the input selector.

**Kirby Version**  
3.3.4

**Additional context**  

https://forum.getkirby.com/t/blueprint-date-field-entering-dates-which-are-10-years-earlier-later/17443

**Comment by afbora** [comment]
@texnixe I tried to fix this issue, but I want to ask something. We can limit the year, but what do we have to do about the days and the months?

For example, if we set the date of `2019-06-15` as the minimum date, we should not be able to choose the day of `2019-06-14`. 

Now even with dropdown, above issue continues. In other words, when we set `2019-06-15` to a minimum date, all days of 2019 can be selected with `dropdown` part. But it shouldn't as expected.

Can we say that we use `minDate` as `minYear`? I'm so confused 🤔
